### ISSUE TITLE
[BUG]: Creation of error report when no network is available

### ISSUE THREAD

**Issue by x51mon** [ankidroid/Anki-Android#14399]
### Checked for duplicates?

- [X] This issue is not a duplicate

### What are the steps to reproduce this bug?

1. put phone into flight mode
2. open AnkiDroid

### Expected behaviour

Use AnkiDroid offline. Don't (automatically) sync if there is no network connection.

### Actual behaviour

Error message: "error sending request for url (): error trying to connect: dns error: failed to lookup address information: No address associated with hostname".
Afterwards AnkiDroid shows a pop-up asking to send an error report to the developer.
Sometimes this repeats several times directly one after the other, before the app can be used.

### Debug info

```text
AnkiDroid Version = 2.16.4 (67fd45a351c44c46c844aa35a402b98a2a487f68)

Android Version = 11

ProductFlavor = full

Manufacturer = unknown

Model = unknown

Hardware = unknown

Webview User Agent = Mozilla/5.0 (Linux; Android 11; ··· Build/RQ3A.211001.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/115.0.5790.136 Mobile Safari/537.36

ACRA UUID = 84202fb8-6cd4-45dc-a504-4b8624f85043

New schema = true

Scheduler = std3

Crash Reports Enabled = true

DatabaseV2 Enabled = true
```

### (Optional) Anything else you want to share?

Already happened on 2.16.2 and 2.16.3.

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by x51mon** [comment]
I'll send the error report from the app referencing this issue

**Comment by BrayanDSO** [comment]
New backend issue, already fixed at 2.17alpha1
